core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5925,6 +5933,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
         = "alluxio.fuse.shared.caching.reader.enabled";
     public static final String FUSE_LOGGING_THRESHOLD = "alluxio.fuse.logging.threshold";
     public static final String FUSE_MAXWRITE_BYTES = "alluxio.fuse.maxwrite.bytes";
+    public static final String FUSE_UMOUNT_WAITTIME =
```

Comment:
I think the part `waittime` is not standard? Could this just be called `alluxio.fuse.umount.timeout`? That name looks like a timeout value for the umount, which is what we want right?

Reply:
Make sense. It's a timeout then waittime. Modified to timeout.